config/provisioners/gcppubsub/gcppubsub.yaml

```diff
@@ -0,0 +1,231 @@
+# Copyright 2018 The Knative Authors
```

Comment:
should this be named `(gcp)pubsub-channel.yaml`? That's what Kafka and In-Memory are named

Reply:
I could be convinced otherwise, but my reasoning is:

`in-memory-channel.yaml` is incorrectly named. I wrote that file when Provisioners were going to support both Sources and Channels. Now that it is only Channels, I don't think we need to specify channel in the name anymore. My (unsubstantiated) guess is that `kafka-channel.yaml` was named based on `in-memory-channel.yaml`.

I think the more important question is whether the `ClusterChannelProvisioner`s should have `-channel` in the name. The two current ones are `in-memory-channel` and `kafka-channel`, whereas this PR has `gcp-pubsub` (as opposed to `gcp-pubsub-channel`).

Reply:
It was named `kafka-provisioner.yaml`, and got recently renamed. @grantr suggested it, due to the fact that there is a `in-memory-channel.yaml` file :smile: 

> I think the more important question is whether the `ClusterChannelProvisioner`s should have `-channel` in the name. 

Actually, I was a bit annoyed by that recently too.  So, my thought now, and it comes back based on your comment here, is that they should be _NOT_ having the `-channel` in their names.

* `kafka(-ccp)`
* `in-memory(-ccp)`

and perhaps `gcp-pubsub-ccp` ? (not sure they really need the `ccp` suffix)

I am happy to take an action item, in renaming them, before we do a 0.2.1 - I guess, this might be backward comp. issue... :thinking: